### PR TITLE
enable website examples to be copied out and run standalone

### PR DESCRIPTION
**Pull request by XiaokaiUber**
For #2136 

#### Background
Each of the website example folders can be copied outside of deck.gl repo and run standalone

#### Change List
- added an index.html template to each folder
- modified app and config files to enable examples to run standalone

### CODE REVIEW

**Comment by Pessimistress** [comment]
`renderToDOM` cannot be called in each app.js. These files are imported by the website.

File: examples/website/3d-heatmap/webpack.config.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const CONFIG = {
 
   // Optional: Enables reading mapbox token from environment variable
   plugins: [
-    new HtmlWebpackPlugin({template: '../template.html'}),
+    new HtmlWebpackPlugin({template: 'index.html'}),
```
Comment (Pessimistress) [inline examples/website/3d-heatmap/webpack.config.js]: Remove this and the html-webpack-plugin dependency. `index.html` can be used without any plugin.

File: examples/website/arc/index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>deck.gl Example</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+    body {margin: 0; font-family: sans-serif; width: 100vw; height: 100vh; overflow: hidden;}
+    .tooltip {pointer-events: none; position: absolute; z-index: 9; font-size: 12px; padding: 8px; background: #000; color: #fff; min-width: 160px; max-height: 240px; overflow-y: hidden;}
+    .tooltip.interactive {pointer-events: all; background: #fcfcfc; color: #444; overflow-y: auto;}
```
Comment (Pessimistress) [inline examples/website/arc/index.html]: If I recall correctly, the `interactive` class name is only used by the icon example. Remove it from others.

File: examples/website/3d-heatmap/webpack.config.js (hunk 1 above)
Comment (Pessimistress) [reply examples/website/3d-heatmap/webpack.config.js]: Remove the dependency from package.json

Comment (XiaokaiUber) [reply examples/website/3d-heatmap/webpack.config.js]: @Pessimistress Updated

**Pull request merged**